elasticsearch.md

```diff
@@ -799,11 +799,108 @@ So that Kibana will forward the necessary headers to Elasticsearch.
 
 [Impersonation](details/impersonation.md) is supported by this rule without an extra configuration.
 
-#### `groups_or` (or `groups`)
+#### Groups subrules
```

Comment:
Groups rules

Reply:
Ad 2 - modified occurrences of groups_and/groups_or/groups all over the documentation - new changes that need to be reviewed.